delft/sequenceLabelling/trainer.py

```diff
@@ -1,30 +1,19 @@
 import os
 
 import numpy as np
+import tensorflow as tf
 from tensorflow.keras.callbacks import Callback, EarlyStopping, ModelCheckpoint
-
-from tensorflow.keras.optimizers import Adam
-from tensorflow.keras.callbacks import Callback, TensorBoard, EarlyStopping, ModelCheckpoint
-from tensorflow.keras.utils import plot_model
-
-from delft.sequenceLabelling.evaluation import accuracy_score, get_report, compute_metrics
-from delft.sequenceLabelling.evaluation import classification_report
-from delft.sequenceLabelling.evaluation import f1_score, accuracy_score, precision_score, recall_score
+from transformers import create_optimizer
```

Comment:
could you add back `plot_model` ? this is in comment, but useful to uncomment when we want a diagram of the architecture.